src/server/hueevents.rs

```diff
@@ -0,0 +1,83 @@
+use std::collections::VecDeque;
+
+use chrono::{DateTime, Utc};
+use tokio::sync::broadcast::Sender;
+
+use crate::hue::event::EventBlock;
+
+#[derive(Clone, Debug)]
+pub struct HueEventRecord {
+    timestamp: DateTime<Utc>,
+    index: u32,
+    pub block: EventBlock,
+}
+impl HueEventRecord {
```

Comment:
Missing newline. It's a tiny detail, but it irks me :sweat_smile:

Reply:
Maybe rustfmt should be ran as part of the CI? Not sure if it would catch this, but might make some future nitpicks unnecessary

Reply:
I've often thought about that. I think it actually allows concatenated items, but it would condense multiple blank lines into a single, at least :)